### DISCUSSION TITLE
BMAX Mini PC B2S e HASSIO 11.1: Errore Bluetooth

### DISCUSSION THREAD

**Discussion by andybegh** [indomus/forum#2683] [HUB: Home Assistant]
Ciao a tutti.
Ho un problema con il mio BMAX B2S e HASSIO 11.1.
Di punto in bianco ha smesso di funzionare il controller bluetooth interno e non mi rileva più i dispositivi... 

<img width="411" alt="Screenshot 2023-11-02 alle 20 31 56" src="https://github.com/indomus/forum/assets/65513270/dfffbf26-340b-4736-a27d-30c757505051">

Inoltre, non mi vede più nemmeno HCI0 da hardware... come posso risolvere?

**Comment by NuccioT** [comment]
Ciao, anch'io ho lo stesso problema! Il fatto è che ho due di queste macchine con due diverse istanze di HA. Su una funziona, sull'altra no! Hai per caso risolto?
